src/interfaces/http/controllers/hero/heroController.js

```diff
@@ -8,17 +9,37 @@ const indexRoute = curry((getAllHeroes, req, res, next) => getAllHeroes().fork(
   ({ result }) => resolveAndSend(res, Status.OK, result),
 ));
 
+const getRoute = curry((getHero, req, res, next) => getHero(getProp('id')(req.params)).fork(
+  ({ error }) => executeErrorHandler(error, res, next),
+  ({ result }) => resolveAndSend(res, Status.OK, result),
+));
+
 const createRoute = curry((createHero, req, res, next) => createHero(req.body).fork(
   ({ error }) => executeErrorHandler(error, res, next),
   ({ result }) => resolveAndSend(res, Status.OK, result),
 ));
 
+const updateRoute = curry((updateHero, req, res, next) => updateHero(getProp('id')(req.params), req.body).fork(
+  ({ error }) => executeErrorHandler(error, res, next),
+  ({ result }) => resolveAndSend(res, Status.OK, result),
+));
+
+const deleteRoute = curry((deleteHero, req, res, next) => deleteHero(getProp('id')(req.params)).fork(
+  ({ error }) => executeErrorHandler(error, res, next),
+  ({ result }) => resolveAndSend(res, Status.OK, result),
```

Comment:
Status.ACCEPTED

And need another method instead of resolveAndSend, the data isn't sent in a delete route.

Reply:
ACCEPTED is better used for async operations and this is not the case.

https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/202